src/components/Footer/Footer.js

```diff
@@ -0,0 +1,33 @@
+import styles from './Footer.module.css'
+import { Icon } from '@iconify/react'
+const Footer = () => {
+  return (
+    <footer className={styles.wrapper}>
+        <div className={styles.content}>
+        <div className={styles.logoAndTime}>
+            <img src="" alt="MEGASTORE" />
+            <p>Пн-Нд 09:00-20:00</p>
+        </div>
+        <div className={styles.info}>
+            <ul>Інформація про компанію
+                <li><a href="#">Про нас</a></li>
+                <li><a href="#">Умови використання сайту</a> </li>
+                <li><Icon className={styles.icon} icon={"fluent:call-12-filled"}/> <a href="tel:+380939475397">+380 939 475 397</a></li>
+                <li><Icon className={styles.icon} icon={"uiw:mail"}/> <a href="mailto:megamarket@market.com">megamarket@market.com</a></li>
```

Comment:
to make the code look cleaner, it is better to write List using .map

Reply:
done